### PR TITLE
core/mpd/local: Add title to get_distinct field types

### PR DESCRIPTION
**Pull request by adamcik**

### CODE REVIEW

**Comment by ZenithDK** [comment]
Shouldn't there be some tests for this also? I wonder how we removed support for title without any tests ever failing?

**Comment by jodal** [comment]
I think it's a bad solution to just add the "title" name from MPD directly. For example, in our search queries, we call this field "track_name". IIRC, we have a draft in a Google Doc for a new set of common names for this.

**Comment by adamcik** [comment]
Mhm, just did this as a quick placeholder name as I was being lazy yesterday

**Comment by adamcik** [comment]
Actually just `track` would be more consistent with the other distinct field names we currently use.

**Comment by adamcik** [comment]
Added a basic test and renamed title to track, but can always change it if need be.

**Comment by jodal** [comment]
`track_name` used to be `track`, but was renamed for consistency with `track_no`.

**Comment by adamcik** [comment]
Fixed.

File: docs/changelog.rst [hunk 1]
```diff
@@ -10,10 +10,12 @@ v1.0.6 (unreleased)
 
 Bug fix release.
 
+- Core/MPD/Local: Add support for ``title`` in
```
Comment (jodal) [inline docs/changelog.rst]: `title` here needs to be updated.

Comment (adamcik) [inline docs/changelog.rst]: Amended to be correct.

**Comment by adamcik** [comment]
Re-updated to not include the ``track_name`` change per IRC.

File: docs/changelog.rst (hunk 1 above)
Comment (jodal) [inline docs/changelog.rst]: Now it needs to be changed to `track` ;-)

Comment (jodal) [inline docs/changelog.rst]: Uhm, from MPD context this changelog entry makes sense. Ignore me.

**Pull request merged**